x/sp/keeper/slash.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func (k Keeper) Slash(ctx sdk.Context, spAcc sdk.AccAddress, rewardInfos []types.RewardInfo) error {
```

Comment:
Shall we use Id in this interface? Not sure, there are other similar interfaces.

Reply:
Fixed.